docs/TileLink-Diplomacy-Reference/index.rst

```diff
@@ -0,0 +1,29 @@
+TileLink and Diplomacy Reference
+================================
+
+TileLink is the cache coherence and memory protocol used by RocketChip and
+other Chipyard generators. It is how different modules like caches, memories,
+peripherals, and DMA devices communicate with each other.
+
+RocketChip's TileLink implementation is built on top of Diplomacy, a framework
```

Comment:
Is it worth adding a link to Henry's CARRV paper/slides about diplomacy? (for those who want something formal)